docs/source/user-guide/guide_indexes.rst

```diff
@@ -94,41 +94,41 @@ Indexes
 5. Note that after saving an index field, an informational table will appear listing the index fields you have created thus far and their names and length.
 
   .. image:: images/indexes/add-index-fields-informational-table.png
     :alt: A portion of the data dictionary creation admin page with a focus on informational table that appears after saving an index field.
 
  a. You also have the ability to edit or modify the fields you've created so far by clicking the gear icon on the right-hand side of this table.
- b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index field. You can then modify this information and either: "Save Index Field" (retain the modifications), "Cancel Index Field" (close the editable form box without retaining any modifications) or "Delete Index Field" (remove the respective index field entirely).
+ b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index field. You can then modify this information and either: "Save index field edit" (retain the modifications), "Cancel index field edit" (close the editable form box without retaining any modifications) or "Remove field from index" (remove the respective index field entirely).
 
     .. image:: images/indexes/edit-index-fields.png
       :alt: A portion of the data dictionary creation admin page with a focus on editable form that appears after you click the gear icon to edit a specific index field.
 
-6. You can enter additional index fields by clicking "Add field" (in this index section) and repeating steps 2-3 in this Index Fields section again.
-7. After defining the index fields, click "Submit Index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
+6. You can enter additional index fields by clicking "Add field to index" (in this index section) and repeating steps 2-3 in this Index Fields section again.
+7. After defining the index fields, click "Save index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
    
   .. image:: images/indexes/edit-index-box.png
-    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Submit Index" button.
+    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Save index" button.
 
 8. Note that after submitting, an informational table will appear listing the indexes you have created thus far and their names, types, and fields (field name and field length).
```

Comment:
8. Note that after saving the index, an informational table will appear...

(The button is "save index" so let's be consistent)